charts/index.yaml

```diff
@@ -1,9 +1,24 @@
 apiVersion: v1
 entries:
   aad-pod-identity:
+  - apiVersion: v1
+    appVersion: 1.7.2
+    created: "2021-02-01T15:06:49.769812-08:00"
+    description: Deploy components for aad-pod-identity
+    digest: 7bf94f8de16675c9a82f32076f1ceacbe4b841d728e215bbc923e4ea9bd4b67f
+    home: https://github.com/Azure/aad-pod-identity
+    maintainers:
+    - email: anish.ramasekar@gmail.com
+      name: aramase
+    name: aad-pod-identity
+    sources:
+    - https://github.com/Azure/aad-pod-identity
+    urls:
+    - https://raw.githubusercontent.com/Azure/aad-pod-identity/master/charts/aad-pod-identity-3.0.1.tgz
+    version: 3.0.1
```

Comment:
Run these steps to generate the charts index:
```bash
mkdir -p ./charts/tmp
helm package ./charts/aad-pod-identity -d ./charts/tmp
helm repo index ./charts/tmp --url https://raw.githubusercontent.com/Azure/aad-pod-identity/master/charts/ --merge ./charts/index.yaml
mv ./charts/tmp/* ./charts
rm -rf ./charts/tmp
```

This will ensure we don't update the `created` date for all the older releases.

Reply:
can we have this as part of CI or use an action like https://github.com/stefanprodan/helm-gh-pages to generate the charts?

Reply:
Opened https://github.com/Azure/aad-pod-identity/issues/956